src/docs/development/platform-integration/ios-app-clip.md

```diff
@@ -58,6 +58,10 @@ Select `Storyboard` for Interface.
 
 Select `UIKit App Delegate` for Life Cycle.
 
+Select the same language as your original target for Language.
+
+(Note: Don't create a Swift target for an Objective-C's project, and vice versa.)
```

Comment:
`(In other words, don't create a Swift App Clip target for an Objective-C main target, and vice versa to simplify the setup.)